external/PATCH-01-STRATUM

```diff
@@ -3912,9 +3918,11 @@ index 9f4bb3a9..e0154d94 100644
 +      RETURN_IF_TDI_ERROR(bf_status);
 +  }
 +
-+  return ::util::OkStatus();
-+}
-+
+   return ::util::OkStatus();
+ }
+ 
+-::util::Status BfSdeWrapper::DeletePort(int device, int port) {
```

Comment:
I see lot of changes in stratum patch around Add and DeletePort and BF vs TDI. Please ensure that you are applying your patch on latest version

Reply:
I just checked it I don't see anything after doing git pull